cmd/state-svc/test/integration/svc_int_test.go

```diff
@@ -207,40 +208,61 @@ func (suite *SvcIntegrationTestSuite) TestAutostartConfigEnableDisable() {
 	ts := e2e.New(suite.T(), false)
 	defer ts.Close()
 
+	autostartDir := filepath.Join(ts.Dirs.Config, "autostart")
```

Comment:
I think you should use something under `ts.Dirs.Work`. Config is a curated directory which autostart is not part of.